prive/utils/data.py

```diff
@@ -127,4 +127,38 @@ def index_split(max_index, split_size, num_splits):
     np.random.shuffle(index_array)
     indices += [index_array[split_size*i : split_size*(i+1)] for i in range(remainder_splits)]
 
-    return indices
\ No newline at end of file
+    return indices
+
+def get_dtype(representation, column_type):
+    """
+
+    Based on the json file schema from the dataset module map type/representation to pandas types.
+    See data-schema-doc.rst documentation for more details.
+
+    Parameters
+    ----------
+    representation: object
+        Representation of column based on the schema this can be a string, and integer, or a list
+    column_type: str
+        String referring to abstract type of the data column (e.g. countable/ordered, finite, etc).
+
+    Returns
+    -------
+    dtype
+        A type for the given column.
+
+    """
+    if representation == 'integer':
+        return np.int
+    elif representation == 'number':
+        return np.float
+    elif representation == 'date':
+        return pd.datetime
+    elif isinstance(representation, (list, str)):
```

Comment:
What is the `str` meant to capture here? I just tried setting up a `pd.CategoricalDtype` in my interpreter with `'12345'` as the categories and it did not work.

Reply:
When the type is either `"finite"` or `"finite/ordered"`, and `"representation"` is a list of string, then the possbile values of that type are the entries in the given list (possibly ordered).

However, I need to figure out how to code this exactly. The "type" is how prive ought to think of the data when it's loaded; whereas "representation" is more about how it's represented in the csv. Those are not _quite_ the same.